src/document/proxy/object_proxy.ts

```diff
@@ -30,11 +30,25 @@ import { RGATreeSplit } from '@yorkie-js-sdk/src/document/json/rga_tree_split';
 import { PlainText } from '@yorkie-js-sdk/src/document/json/plain_text';
 import { RichText } from '@yorkie-js-sdk/src/document/json/rich_text';
 import { ArrayProxy } from '@yorkie-js-sdk/src/document/proxy/array_proxy';
-import { TextProxy } from '@yorkie-js-sdk/src/document/proxy/text_proxy';
-import { RichTextProxy } from '@yorkie-js-sdk/src/document/proxy/rich_text_proxy';
+import { TextProxy, TText } from '@yorkie-js-sdk/src/document/proxy/text_proxy';
+import {
+  RichTextProxy,
+  TRichText,
+} from '@yorkie-js-sdk/src/document/proxy/rich_text_proxy';
 import { toProxy } from '@yorkie-js-sdk/src/document/proxy/proxy';
 import { CounterType, Counter } from '@yorkie-js-sdk/src/document/json/counter';
-import { CounterProxy } from '@yorkie-js-sdk/src/document/proxy/counter_proxy';
+import {
+  CounterProxy,
+  TCounter,
+} from '@yorkie-js-sdk/src/document/proxy/counter_proxy';
+
+export type TObject<T = unknown> = {
+  getID?(): TimeTicket;
+  toJSON?(): string;
+  createText?(key: string): TText;
+  createRichText?(key: string): TRichText;
+  createCounter?(key: string, value: CounterType): TCounter;
```

Comment:
It seems that the 3 methods below should be created as a new object later, not as an object method.
```ts
createText?(key: string): TText;
createRichText?(key: string): TRichText;
createCounter?(key: string, value: CounterType): TCounter;
```

ex)
```ts
root['k1'] = new Text('');
root['k2'] = new RichText(''');
```

Reply:
It would be good to work on this in a subsequent PR.